src/sage/combinat/posets/hasse_diagram.py

```diff
@@ -1160,25 +1158,29 @@ def coxeter_transformation(self, algorithm='cython'):
 
         EXAMPLES::
 
-            sage: P = posets.PentagonPoset()._hasse_diagram                             # optional - sage.modules
-            sage: M = P.coxeter_transformation(); M                                     # optional - sage.libs.flint sage.modules
+            sage: # needs sage.modules
+            sage: # needs sage.libs.flint sage.modules
+            sage: P = posets.PentagonPoset()._hasse_diagram
+            sage: M = P.coxeter_transformation(); M
             [ 0  0  0  0 -1]
             [ 0  0  0  1 -1]
             [ 0  1  0  0 -1]
             [-1  1  1  0 -1]
             [-1  1  0  1 -1]
-            sage: P.__dict__['coxeter_transformation'].clear_cache()                    # optional - sage.libs.flint sage.modules
-            sage: P.coxeter_transformation(algorithm="matrix") == M                     # optional - sage.libs.flint sage.modules
+            sage: P.__dict__['coxeter_transformation'].clear_cache()
+            sage: P.coxeter_transformation(algorithm="matrix") == M
             True
 
         TESTS::
 
-            sage: P = posets.PentagonPoset()._hasse_diagram                             # optional - sage.modules
-            sage: M = P.coxeter_transformation()                                        # optional - sage.libs.flint sage.modules
-            sage: M**8 == 1                                                             # optional - sage.libs.flint sage.modules
+            sage: # needs sage.modules
```

Comment:
This line is redundant because of the next line

Reply:
Done.